test/invariant/actors/BadVoter.t.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.19;
+
+import { BaseActor, IERC20, IERC20Metadata, AngleGovernor, TestStorage } from "./BaseActor.t.sol";
+import { console } from "forge-std/console.sol";
+import { ProposalStore, Proposal } from "../stores/ProposalStore.sol";
+import { IGovernor } from "oz/governance/IGovernor.sol";
+
+contract BadVoter is BaseActor {
+    AngleGovernor internal _angleGovernor;
+    ProposalStore public proposalStore;
+
+    constructor(
+        AngleGovernor angleGovernor,
+        IERC20 _agToken,
+        uint256 nbrVoter,
+        ProposalStore _proposalStore
+    ) BaseActor(nbrVoter, "BadVoter", _agToken) {
+        _angleGovernor = angleGovernor;
+        proposalStore = _proposalStore;
+    }
+
+    function vote(uint256 actorIndexSeed, uint256 proposalId) public useActor(actorIndexSeed) {
```

Comment:
Ok so for now we don't do anything with this one, so what I think we can do (they are not very much realistic as tested in the unit and fuzz, but good practice):

- try to execute while it is not in the succeeded state
- try to schedule the proposal dircetly on the timelock when it is created
- add some delegation and try to double vote
- ...